gateway/gateway.go

```diff
@@ -38,6 +40,23 @@ var defaultQueryFrontendAPIs = []string{
 	"/api/prom/api/v1/status/buildinfo",
 }
 
+var defaultAlertManagerAPIs = []string{
+	"/alertmanager",
+	"/api/prom",
```

Comment:
```suggestion
	"/api/prom/alertmanager/",
```
But needs testing. I think the docs are wrong https://cortexmetrics.io/docs/api/#alertmanager-ui

Reply:
How can I test this endpoint?

Reply:
I would start cortex with target=alertmanager and see what paths are available in cortex

Reply:
You are right, the documentation is wrong. I got 404 when I requested `/api/prom` endpoint. However, `/api/prom/alertmanager` does not work either. I got 404 when I requested that endpoint, too. 

Everything that is related to alertmanager ui is under `/alertmanager/`. I propose to remove `/api/prom` altogether and only implement `/alertmanager/`. If we come to an agreement, I will open a PR to fix the documentation as well

Reply:
just finish this PR then with /alertmanager only.